components/learn/platform/Platform.js

```diff
@@ -282,7 +282,7 @@ export default function Platform(props) {
           </div>
 
           <div className={styles.pGroup}>
-            <h3>Ballerina persist</h3>
+            <h3>Bal persist</h3>
```

Comment:
```suggestion
            <h3>`bal persist`</h3>
```

Reply:
> ```suggestion
> 
>             <h3>`bal persist`</h3>
> 
> ```

As this a header, Let's gone with Bal persist. We have the same in the BBEs as well

Reply:
Still, we need backticks.